ports/libsimpleservo/api/src/lib.rs

```diff
@@ -488,7 +488,8 @@ impl ServoGlue {
                             .host_callbacks
                             .on_allow_navigation(url.to_string());
                         let window_event = WindowEvent::AllowNavigationResponse(pipeline_id, data);
-                        let _ = self.process_event(window_event);
+                        self.events.push(window_event);
```

Comment:
Perhaps add a `FIXME` here to say that this should go ideally through `host_callbacks`?

Reply:
What do you mean go through host_callbacks?

Reply:
If we wanted to actually ask the user for permission, it would have to go through there, right?

Reply:
Isn't that what's happening two lines earlier?

Reply:
Oops...